eng/WorkaroundsImported.targets

```diff
@@ -0,0 +1,5 @@
+<!-- Use this file to workaround issues. List the issue tracking the item to fix so we can remove the workaround when the issue is resolved. -->
+<Project>
+  <!-- Workaround https://github.com/dotnet/source-build/issues/1112. Source link is currently disabled in source build so define this dummy target which is required for pack. -->
+  <Target Name="InitializeSourceControlInformation"/>
```

Comment:
This is an empty target in Microsoft.Common.CurrentVersion.targets. Why does it need to be defined again here?

Reply:
It's normally a target defined by in source link. I see that in a few place in our code we have custom targets that depends on this target or reference it in Before/AfterTarget during pack. Since sourcelink is currently disabled, we need to define a this target so that those custom targets still work.